static/styles/subscriptions.css

```diff
@@ -241,7 +241,7 @@ form#add_new_subscription {
     width: calc(100% - 15px);
 }
 
-#stream_name_error {
+.stream_creation_error {
```

Comment:
I am concerned about this renaming. A quick `git grep` shows that `stream_name_error` is used about a dozen times throughout the codebase. Also, a `stream_name_error` seems very different from a `stream_creation_error`, wouldn't it make sense to keep these both separate instead of renaming?

Reply:
To apply same style/css to all stream creation error, I have rename it to `stream_creation_error` and add class `stream_creation_error` to all error which are shown in stream create page.(like `stream_name_error` and `stream_subsription_error`) It also helps to add more error element with different IDs and same class, so that same style will be applied to all error element. 
I have not changed div element id.

Reply:
Ah okay, makes sense! My bad! Thanks for explaining! :)